### PR TITLE
Feature/theme composer util

### PR DESCRIPTION
**Pull request by barneyfoxuk**
# Description

# Checklist

Pull request contains:

- [ ] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

File: src/utils/theme-composer/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  '@uswitch/trustyle.hero',
+  '@uswitch/trustyle.input',
+  '@uswitch/trustyle.list',
+  '@uswitch/trustyle.pagination',
+  '@uswitch/trustyle.toggle-switch'
+]
+const COMPOUNDS = [
+  '@uswitch/trustyle.article-intro',
+  '@uswitch/trustyle.side-nav'
+]
```
Comment (callumacrae) [inline src/utils/theme-composer/src/index.ts:27]: Can't you look these up in the package.json files?

Comment (barneyfoxuk) [reply src/utils/theme-composer/src/index.ts:27]: there's no mapping for what's an element and what's a compound

Comment (barneyfoxuk) [reply src/utils/theme-composer/src/index.ts:27]: (they are looked up in package.json - this assigns them to the correct group)

Comment (callumacrae) [reply src/utils/theme-composer/src/index.ts:27]: ahh right because they're coming from node_modules, not this project - got it.

Comment (callumacrae) [reply src/utils/theme-composer/src/index.ts:27]: Could also add it as another field in package.json

Comment (barneyfoxuk) [reply src/utils/theme-composer/src/index.ts:27]: Yeah or if the package names contained it, i.e. `@uswitch/trustyle.element.input`

**Pull request merged**